docker/README.md

```diff
@@ -13,17 +13,17 @@ Requires a running deployment of registry
 
 To build and run  (assuming registry local-dev defaults for host/credentials)
 
-    cd path/to/registry-sweepers
-    docker build -t registry-sweepers .
+    cd path/to/registry-sweepers/
+    docker build -t registry-sweepers docker --file ./docker/Dockerfile .
     docker run -e PROV_ENDPOINT='https://localhost:9200/' -e PROV_CREDENTIALS='{"admin": "admin"}' registry-sweepers
```

Comment:
I'd like to see `--env` instead of `-e` and the full `docker container run` but that's just my own preference 😊

Reply:
ok, that works for me